### PR TITLE
Feat/follow unfollow and likes functionality

### PR DESCRIPTION
**Pull request by Maianki**
### Added like, follow and unfollow functionalilty

This PR includes
- [ ] follow functinality
- [ ] unfollow functionality
- [ ] like funcitonality
- [ ] filters

### CODE REVIEW

**Review by anik31 — APPROVED** [review]
The follow/unfollow & likes functionality is working fine. Great work.

File: src/features/Post/postSlice.js [hunk 1]
```diff
@@ -132,10 +131,46 @@ export const deleteComment = createAsyncThunk(
   }
 );
 
+export const likePost = createAsyncThunk(
+  "post/likePost",
+  async ({ token, postId }, { rejectWithValue }) => {
+    try {
+      const response = await likePostService(token, postId);
+
+      console.log(response.data.posts);
```
Comment (vrx29) [inline src/features/Post/postSlice.js]: remove unnecessary console statements

**Review by vrx29 — APPROVED** [review]
Everything looks good. You can merge it.

File: src/features/Post/postSlice.js (hunk 1 above)
Comment (Maianki) [reply src/features/Post/postSlice.js]: sure will do that

**Pull request merged**